hasher-matcher-actioner/hma-lite/API.md

```diff
@@ -0,0 +1,29 @@
+Here's a super-rough first cut for an API
+
+# Match a single hash against the index
+
+```
+GET /v1/hashes/query?hash=hash_in_hex_string
+```
+
+`200 OK` if found, `404 Not Found` if not.
```

Comment:
LOL @ 404.

However, worried this is more confusing than a 200 with a json blob sent back, but don't think it's too different, and we can get feedback from externals first.